### PR TITLE
feat: Added Python Official Docs Details in resources Section

### LINKED ISSUE #1385: [Add] : Add Python Official Docs in Official Docs Sub Category in Resources Main Category
### Site URL

https://www.python.org/doc/

### Description of the above provided site

- Add Python Official Docs in Official Docs Sub Category in Resources Main Category

### Category

Resources

### Sub Category

official-docs

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by NarendraKoya999**
## Fixes Issue 

Closes #1385 

## Changes proposed
- **Added Python Official Docs Details in officialdocs Sub Category in resources category.**

### CODE REVIEW

File: database/resources/official-docs.json [hunk 1]
```diff
@@ -96,6 +96,13 @@
     "url": "https://guides.emberjs.com/release/",
     "category": "resources",
     "subcategory": "officialdocs"
+  },
+  {
+    "name": "Python(Official Doc)",
+    "description": "Python's documentation, tutorials, and guides are constantly evolving.Get started here, or scroll down for documentation broken out by type and subject.",
```
Comment (CBID2) [inline database/resources/official-docs.json:102]:
```suggestion
    "description": "Python's documentation, tutorials, and guides are constantly evolving.  Get started here, or scroll down for documentation categorized by type and subject.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]

**Review by CBID2 — APPROVED** [review]
LGTM

File: database/resources/official-docs.json [hunk 2]
```diff
@@ -96,6 +96,13 @@
     "url": "https://guides.emberjs.com/release/",
     "category": "resources",
     "subcategory": "officialdocs"
+  },
+  {
+    "name": "Python(Official Doc)",
+    "description": "Python's documentation, tutorials, and guides are constantly evolving.  Get started here, or scroll down for documentation categorized by type and subject.",
```
Comment (rupali-codes) [inline database/resources/official-docs.json:102]:
```suggestion
    "description": "It's the official documentation of Python. Start your journey here, or explore categorized resources covering language basics, libraries, web development, data science, AI, and more. ",
```

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Pull request merged**